landscape/client/broker/exchange.py

```diff
@@ -642,6 +651,7 @@ def schedule_exchange(self, urgent=False, force=False):
                 interval = self._config.urgent_exchange_interval
             else:
                 interval = self._config.exchange_interval
+            interval += self._backoff_counter.get_random_delay()
```

Comment:
shouldn't you pass self._config.stagger here to use the same random ratio as configured, instead of the arbitrary ratio you have?

Reply:
In this specific case, although 25% is an arbitrary random stagger, I don't think it's a good idea of the user to control this value, as it's more of an implementation detail, which its only purpose is so that requests don't hit the server at the same time if it is under heavy load. However you bring up a good point, since none of this is not currently configurable, maybe we should let the the start and stop be configurable?

Reply:
The config default value is pretty arbitrary too. However, I think you're right: There is no need to have this as user-configurable since it shouldn't stagger under normal circumstances and, unlike the stagger configuration, you can just bypass any delay by restarting the client.

Ignore my previous comment.